addons/l10n_es_pos/models/pos_order.py

```diff
@@ -0,0 +1,61 @@
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    is_l10n_es_simplified_invoice = fields.Boolean(
+        "Simplified invoice",
+        copy=False,
+        default=False,
+    )
+    l10n_es_unique_id = fields.Char(
+        "Simplified invoice number",
+        copy=False,
+    )
+
+    @api.model
+    def _simplified_limit_check(self, amount_total, limit):
+        precision_digits = self.env["decimal.precision"].precision_get("Account")
+        # -1 or 0: amount_total <= limit, simplified
+        #       1: amount_total > limit, can not be simplified
+        return float_compare(amount_total, limit, precision_digits=precision_digits) < 0
```

Comment:
The comment say that if the amount_total is less or equal, the order is simplified and if it is strictly greater, the order is not. The code says that amount_total is strictly smaller, the order is simplified and if it is greater or equal, the order is not. This is a small difference but if you can align the comment and the code that would be nice.

Reply:
you are right. I removed the comment completely. I think that in this case it adds more confusion that anything else